learn/sherif/main_test/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -26,26 +28,37 @@
  */
 public class RobotContainer {
     // The robot's subsystems and commands are defined here...
-    private Factory m_f = new Factory();
-    private DriveTrain m_drivetrain;
-    private Joystick m_controller;
+    public Factory factory = new Factory();
+    private DriveTrain m_drivetrain = null;
+    private Joystick m_controller = null;
```

Comment:
Also there will eventually be 2 joysticks, IDs 0 and 1

Reply:
I'll add that when we need to use the second.

Reply:
ok sounds good